doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -28,7 +28,11 @@ To enable a PSA driver, set the following configurations:
 
 If multiple drivers are enabled, the first ordered item in this table takes precedence for an enabled cryptographic feature, unless the driver does not enable or support it.
 
-Enabling or disabling PSA driver specific configurations controls the support for a given algorithm, per driver.
+The configuration granularity of the driver :ref:`nrf_security_drivers_cc3xx` allows enabling or disabling of specific PSA APIs (such as psa_cipher_encrypt, psa_sign_hash), but not individual algorithms.
+
+The driver :ref:`nrf_security_drivers_oberon` allows higher configuration granularity, allowing you to enable or disable individual algorithms as well.
+
+When multiple drivers are enabled and support the same cryptographic feature, the configuration tries to minimize the code size and, when possible, only include one implementation.
```

Comment:
```suggestion
When multiple drivers are enabled and support the same cryptographic feature, the configuration system tries to minimize the code size and, when possible, only include one implementation.
```

Also there is something off with the tenses.

AI gives this phrasing, in case you like it:

"When multiple drivers are enabled and support the same cryptographic feature, the configuration system attempts to minimize the code size by including only one implementation, whenever possible"

But it's not clear to me if those are the semantics you are trying to communicate.

Reply:
```suggestion
"When multiple enabled drivers supports the same cryptographic feature, the default behavior is to include only one implementation to minimize code size."
```

Reply:
I kind of combined these two suggestions in something new.